test/repos/lots_of_diffs.sh

```diff
@@ -1,35 +0,0 @@
-#!/bin/bash
```

Comment:
it looks like this file covers more test cases than is covered in this PR

Reply:
To know if a test cover a function you have to look at codecov, https://codecov.io/gh/jesseduffield/lazygit/src/master/pkg/commands/git.go :  
<img width="1154" alt="capture d ecran 2018-09-19 a 11 34 44" src="https://user-images.githubusercontent.com/4854264/45745051-3d580280-bc00-11e8-8493-5687e5bfd02e.png">